### PR TITLE
[C-2877] Address suggested-tracks qa #2

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Another round of qa fixes addressing callouts from design

### CODE REVIEW

File: packages/common/src/utils/updatePlaylistArtwork.ts [hunk 1]
```diff
@@ -76,23 +80,34 @@ export const updatePlaylistArtwork = async (
...
-      tracksForImage.map(async (track) => {
-        const { cover_art_sizes, cover_art } = track
-        return await context.audiusBackend.getImageUrl(
-          cover_art_sizes ?? cover_art,
-          SquareSizes.SIZE_1000_BY_1000
-        )
-      })
-    )
+    if (tracksForImage.length === 0) {
+      console.log('okay full removal!')
```
Comment (Kyle-Shanks) [inline packages/common/src/utils/updatePlaylistArtwork.ts:92]: log

File: packages/common/src/utils/updatePlaylistArtwork.ts [hunk 2]
```diff
@@ -76,23 +80,34 @@ export const updatePlaylistArtwork = async (
...
+      // @ts-expect-error
```
Comment (Kyle-Shanks) [inline packages/common/src/utils/updatePlaylistArtwork.ts:93]: still expecting this error?

File: packages/common/src/utils/updatePlaylistArtwork.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { AudiusBackend } from 'services/audius-backend'
 import { Nullable } from './typeUtils'
 
 const filterTrack = (track1: Track) => (track2: Track) =>
-  track1.track_id === track2.track_id
+  track1.track_id !== track2.track_id
```
Comment (Kyle-Shanks) [inline packages/common/src/utils/updatePlaylistArtwork.ts:11]: lol nice

**Review by Kyle-Shanks — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-suggested-tracks-qa-2

File: packages/common/src/utils/updatePlaylistArtwork.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/common/src/utils/updatePlaylistArtwork.ts:92]: lmao

File: packages/common/src/utils/updatePlaylistArtwork.ts (hunk 2 above)
Comment (dylanjeffers) [reply packages/common/src/utils/updatePlaylistArtwork.ts:93]: yes actually, apparently it cant be nullable?

File: packages/common/src/utils/updatePlaylistArtwork.ts (hunk 3 above)
Comment (dylanjeffers) [reply packages/common/src/utils/updatePlaylistArtwork.ts:11]: yeah shhh

**Pull request merged**